src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,11 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+    /**
+     * A read-only immutable interface for a Person Details in the addressbook.
```

Comment:
Do these comments apply to the `getPrintableString()` method or the `Printable` interface?